webapp/components/Comment.vue

```diff
@@ -28,13 +27,27 @@
       </no-ssr>
       <!-- eslint-disable vue/no-v-html -->
       <!-- TODO: replace editor content with tiptap render view -->
+
+      <div v-if="isCollapsed" v-html="comment.contentExcerpt" style="padding-left: 40px;" />
+      <div
+        v-show="comment.content !== comment.contentExcerpt"
+        style="text-align: right;  margin-right: 20px; margin-top: -12px;"
+      >
+        <a v-if="isCollapsed" style="padding-left: 40px;" @click="isCollapsed = !isCollapsed">
+          {{ $t('comment.show.more') }}
+        </a>
+      </div>
+      <div v-if="!isCollapsed" v-html="comment.content" style="padding-left: 40px;" />
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-24T19:46:37Z" title="Monday, June 24th 2019, 9:46:37 pm +02:00">Jun 24, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -23,10 +23,27 @@
     <!-- eslint-disable vue/no-v-html -->
     <!-- TODO: replace editor content with tiptap render view -->
     <ds-space margin-bottom="small" />
-    <div style="padding-left: 40px;" v-html="comment.contentExcerpt" />
-    <!-- eslint-enable vue/no-v-html -->
+    <div v-if="!isActive" v-html="comment.contentExcerpt" style="padding-left: 40px;" />
+    <div v-show="comment.content !== comment.contentExcerpt">
+      <span
+        v-if="!isActive"
+        style="padding-left: 40px; color: #17b53f; cursor:pointer"
+        @click="isActive = !isActive"
+      >
+        {{ $t('comment.show.more') }}
+      </span>
+    </div>
+    <div v-if="isActive" v-html="comment.content" style="padding-left: 40px;" />
+    <span
+      v-if="isActive"
+      @click="isActive = !isActive"
+      style="padding-left: 40px; color: #17b53f; cursor:pointer"
+    >
+      {{ $t('comment.show.less') }}
```


Could you DRY this out? (DRY == don't repeat yourself, here is a [link](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)). I think if you put the `isCollapsed` property to the root node, that should be it.

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-07-03T06:36:35Z" title="Wednesday, July 3rd 2019, 8:36:35 am +02:00">Jul 3, 2019</time>_

---


Same same … as said before. Not just this …